modules/fedimint-lnv2-tests/bin/tests.rs

```diff
@@ -260,7 +264,18 @@ async fn test_payments(dev_fed: &DevJitFed) -> anyhow::Result<()> {
         await_receive_claimed(&client, receive_op).await?;
     }
 
-    info!("Testing LDK Gateway can pay HOLD invoice...");
+    retry(
```

Comment:
Was there a flake that prompted this change? I would think that `await_receive_claimed` would wait until the balance was been updated, so the retry wouldnt be necessary